Dockerfile

```diff
@@ -21,15 +21,29 @@ RUN mkdir -p $AKAMAI_CLI_HOME/.akamai-cli
 RUN apk add --no-cache docker git bash python2 python2-dev py2-pip python3 python3-dev npm wget jq openssl openssl-dev curl nodejs build-base libffi libffi-dev vim nano util-linux go dep tree bind-tools 
 RUN go get -d github.com/akamai/cli && cd $GOPATH/src/github.com/akamai/cli && go mod init && go mod tidy && go build -o /usr/local/bin/akamai
 RUN pip install --upgrade pip && pip3 install --upgrade pip
+RUN npm config set unsafe-perm true
```

Comment:
Should we have "arcane" commands commented so that we understand why we did this down the line?